packages/salesforcedx-apex-replay-debugger/src/adapter/apexReplayDebug.ts

```diff
@@ -556,10 +627,14 @@ export class ApexReplayDebug extends LoggingDebugSession {
           source: args.source,
           line: bp.line
         });
-        if (isVerified) {
-          this.breakpoints
-            .get(uri)!
-            .push(this.convertClientLineToDebugger(bp.line));
+        if (!isVerified) {
+          this.sendEvent(
+            new Event(SEND_METRIC_ERROR_EVENT, {
+              subject: 'Failed to set breakpoint',
+              callstack: new Error().stack,
+              message: `Failed to set breakpoint at line ${bp.line} in ${args.source.path}`
+            })
+          );
```

Comment:
Why is this behavior removed?  Should it be converted to an if/else statement instead?

Reply:
oops, deleted by mistake